dsl/src/commonMain/kotlin/it/unibo/collektive/networking/NetworkManager.kt

```diff
@@ -0,0 +1,47 @@
+package it.unibo.collektive.networking
+
+import it.unibo.collektive.ID
+import it.unibo.collektive.messages.AnisotropicMessage
+import it.unibo.collektive.messages.IsotropicMessage
+import it.unibo.collektive.messages.ReceivedMessage
+import it.unibo.collektive.messages.SentMessage
+
+/**
+ * Implementation of the Network interface.
+ */
+class NetworkManager {
+    private var messageBuffer = mutableSetOf<SentMessage>()
```

Comment:
I prefer the use of `var` with an immutable collection.
I recommend using mutable collection only in those cases in which there is an evident advantage in its use, otherwise use an immutable one.

Reply:
Ok, I changed messageBuffer into immutable set.